### PR TITLE
Added Doctrine caching

### PR DESCRIPTION
**Pull request by leonverschuren**

### CODE REVIEW

**Comment by rvanlaarhoven** [comment]
@leonverschuren , how does this work on the dev environment? Is there a way to disable caching?

If not, I think we have to build it in this bundle. E.g. add some configuration option for it.

File: Model/ContentRepository.php [hunk 1]
```diff
@@ -224,18 +221,24 @@ public function findAnonymouslyCreatedById($id)
...
+     *
      * @todo just retrieves random content items for now
+     *
+     * @param     $template
+     * @param int $limit
+     *
+     * @return \Doctrine\Common\Collections\ArrayCollection
      */
-    public function findPopularByTemplate($template)
+    public function findPopularByTemplate($template, $limit = 10)
```
Comment (rvanlaarhoven) [inline Model/ContentRepository.php]: While you're at it, could you mark this method as deprecated?

Something like

    @deprecated will be removed in 1.0

File: Model/ContentRepository.php [hunk 2]
```diff
@@ -344,20 +344,23 @@ public function findSortedByTemplateTitle($template, $order = 'ASC')
      */
     public function findRelated(Content $content, $limit = 10)
     {
+        $city = $content->getValueSet()->getValueFor('address')->getAddress()->getCity();
```
Comment (rvanlaarhoven) [inline Model/ContentRepository.php]: I don't think this belongs here... Maybe we should mark this method as deprecated as well. What do you think?

Comment (leonverschuren) [inline Model/ContentRepository.php]: I agree, looks like a project specific function

**Pull request merged**